web/.eslintrc.js

```diff
@@ -1,35 +1,35 @@
 module.exports = {
   root: true,
-  parser: "@typescript-eslint/parser",
+  parser: '@typescript-eslint/parser',
   parserOptions: {
-    "project": "tsconfig.json",
+    project: 'tsconfig.json',
   },
   extends: [
-    "next",
-    "next/core-web-vitals",
-    "eslint:recommended",
-    "plugin:react/recommended",
-    "plugin:tailwindcss/recommended",
-    "plugin:import/recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:@typescript-eslint/recommended-requiring-type-checking",
-    "plugin:prettier/recommended",
+    'next',
+    'next/core-web-vitals',
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:tailwindcss/recommended',
+    'plugin:import/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+    'plugin:prettier/recommended',
   ],
   plugins: [
-    "prettier",
-    "react",
-    "tailwindcss",
-    "import",
-    "simple-import-sort",
-    "@typescript-eslint",
+    'prettier',
+    'react',
+    'tailwindcss',
+    'import',
+    'simple-import-sort',
+    '@typescript-eslint',
   ],
   rules: {
-    semi: "error",
-    "prefer-const": "error",
-    "react/react-in-jsx-scope": "off",
-    "simple-import-sort/imports": "error",
-    "simple-import-sort/exports": "error",
-    "@typescript-eslint/no-unused-vars": "warn",
-    "@typescript-eslint/no-empty-function": "warn",
+    semi: 'error',
+    'prefer-const': 'error',
+    'react/react-in-jsx-scope': 'off',
+    'simple-import-sort/imports': 'error',
+    'simple-import-sort/exports': 'error',
+    '@typescript-eslint/no-unused-vars': 'warn',
+    '@typescript-eslint/no-empty-function': 'warn',
   },
```

Comment:
I'm seeing some styling changes here that don't reflect our config (the single quotation vs double). If you are using prettier alone this might be why. With this project you only need ESLint, prettier was integrated as an ESLint plugin